modules/ec2/variables.tf

```diff
@@ -9,7 +9,6 @@ variable "base_instance_type" {
 variable "base_ami" {
   description = "Value of the base AMI that we're using"
   type        = string
-  default     = "ami-0ecc74eca1d66d8a6"
```

Comment:
do we want to default to new "ami-0267f74ea1e610dec" ? not a blocker

Reply:
the pattern is that we do not specify any defaults in the module's variable files. not sure why but the ami is declared in `main.tf` as a local variable.